supertape.com.website.json

```diff
@@ -0,0 +1,24 @@
+{
+  "providerId": "supertape.com",
+  "providerName": "Supertape",
+  "serviceId": "website",
+  "serviceName": "Supertape site",
+  "version": 1,
+  "logoUrl": "https://svgshare.com/i/15HY.svg",
+  "description": "Enables a domain to work with Supertape",
+  "hostRequired": true,
```

Comment:
Pls consider adding signing. Some DNS providers would require it for onboarding even though in this case the template would not have security concerns.